CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ Note that since we don't clearly distinguish between a public and private interf
     - `native`, no changes
 - Add `CustomProperty.Context.errorContext` to support reporting errors during loading of custom properties (#1254)
   - Use in MolViewSpec extension
+- Add basic support for volumetric data
```

Comment:
this is just for MVS, right?

Reply:
> this is just for MVS, right?

Yes, you are right, I will edit the description to highlight that this is just for MVS. I apologise for missing this point initially.

Reply:
> this is just for MVS, right?

Fixed in ba42726f85b2056013ea5a3e4912d28e604985e8. Thanks again for the catch.